discovery_engine/lib/src/domain/models/feed_market.dart

```diff
@@ -27,11 +27,11 @@ typedef FeedMarkets = Set<FeedMarket>;
 class FeedMarket with _$FeedMarket {
   @HiveType(typeId: feedMarketTypeId)
   const factory FeedMarket({
-    /// Country code as per ISO 3166-1 alpha-2 definition.
-    @HiveField(0) required String countryCode,
-
     /// Language code as per ISO ISO 639-1 definition.
     @HiveField(1) required String langCode,
+
+    /// Country code as per ISO 3166-1 alpha-2 definition.
+    @HiveField(0) required String countryCode,
```

Comment:
❓ i guess we can't change the value of the `HiveField` as well without breaking stuff, right?

Reply:
the field id? no

the field type? yes, but probably not worth it

Reply:
I mean I guess you can change both if you create a new version and some form of adapter etc.